prac_07/myguitars.py

```diff
@@ -0,0 +1,48 @@
+"""
+
+estimation - 30 mins
+actual - 60 mins
+
+"""
+
+import csv
+from collections import namedtuple
+from guitar import Guitar
+from operator import itemgetter
+
+FILENAME = "guitars.csv"
+INDEX_NAME = 0
+INDEX_YEAR = 1
+INDEX_COST = 2
+
+
+def main():
+    """Program that reads and adds guitars to a csv file."""
+    guitars = []
+    with open(FILENAME, "r") as in_file:
+        in_file.readline()
+        reader = csv.reader(in_file)
+        NewGuitar = namedtuple('Guitar', 'name, year, cost')
+        for row in reader:
+            print(Guitar(row[INDEX_NAME], row[INDEX_YEAR], float(row[INDEX_COST])))
+            guitars.append(NewGuitar._make(row))
+
+    guitars.sort(key=itemgetter(INDEX_YEAR))
```

Comment:
I was able to sort with just "sorted(guitars)" so might help the codes simplicty